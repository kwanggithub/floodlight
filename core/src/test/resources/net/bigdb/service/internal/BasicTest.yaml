treespaces:
  - name: BasicTest
    data_sources:
        - name: config
          implementation_class: net.bigdb.data.memory.MemoryDataSource
    data_source_mappings:
      - data_source: config
        predicate: Config
      - data_source: "$data-source"
        predicate: "!Config"
    module_search_paths:
      - path: src/test/resources/net/bigdb/service/internal
        recursive: true
    modules:
      - name: BasicTest
auth_config:
    enable_null_authentication: true
    default_authorization_hook: net.bigdb.auth.NullAuthorizationHook
    session_manager: net.bigdb.auth.session.SimpleSessionManager
