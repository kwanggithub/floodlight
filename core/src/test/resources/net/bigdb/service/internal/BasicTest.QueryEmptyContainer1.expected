[ {
  "child2" : {
    "child21" : {
      "child211" : "foo"
    }
  }
} ]